ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/decoders/InlineTableDecoderTest.kt

```diff
@@ -30,22 +32,78 @@ class InlineTableDecoderTest {
     data class Version(val ref: String)
 
     @Test
-    @Ignore
+    fun decodeSimpleInlineTable() {
+        @Serializable
+        data class NestedTable(
+            val name: String,
+            @SerialName("configurationList")
+            val overriddenName: List<String?> = listOf(),
+        )
+
+        @Serializable
+        data class Table2(
+            @SerialName("akuleshov7.com")
+            val inlineTable: NestedTable,
+        )
+
+        @Serializable
+        data class MyClass(
+            val table2: Table2,
+        )
+
+        val test =
```

Comment:
Hm, it looks like deserialization should go in a little bit different way: https://github.com/toml-lang/toml/discussions/879

```
table2 = { table2."akuleshov7.com" = { name = "test" }, i = 1 }
```

is equal to the following one:
```
[table2]
    i = 1
[table2.table2."akuleshov7.com"]
    name = "test"
```

Reply:
So in `Table2` class there should be another one `table2` field

Reply:
Will be also great if you could add a test which deserializes the following code:
```
[table2]
    i = 1
[table2.table2."akuleshov7.com"]
    name = "test"
```

to same structures

Reply:
Done

Reply:
Thanks, brat! LGTM. Free to merge (don't forget to squash)